decidim-meetings/spec/system/explore_meetings_spec.rb

```diff
@@ -372,7 +372,7 @@
       end
 
       it "works with 'back to list' link" do
-        skip_unless_redesign_enabled
+        skip "REDESIGN_PENDING - Adapt this test after merging https://github.com/decidim/decidim/pull/10376 which implements the back link"
```

Comment:
https://github.com/decidim/decidim/pull/10376 has been merged already.

Reply:
Behavior and test recovered

Reply:
Anyway the test works with redesigns enabled, so I added a `skip_unless_redesign_enabled`